### ISSUE TITLE
Removing child node lets siblings disappear

### ISSUE THREAD

**Issue by maltmann-muc** [mbraak/jqTree#582]
I have nodes and child nodes (one level).
When calling removeNode($childnode), the siblings disappear.

Debugged (jqTree 1.4.11 on Chrome) and found this issue:
- JqTreeWidget.prototype.removeNode() calls node.remove()
- Node.prototype.remove() sets this.parent to null
- Back in JqTreeWidget.prototype.removeNode(),  this._refreshElements(node.parent) is then called with node.parent as null

**Comment by mbraak** [comment]
I couldn't reproduce this. I used this code:

```js
var n = $tree.tree("getNodeByName", "Spinosauroids");
console.log(n);
$tree.tree("removeNode", n);
```

**Comment by mbraak** [comment]
I updated the code so it correctly calls `_refreshElements` with a valid parent. This is fixed in #583.

However, I could not reproduce the error.
